src/tests/ftest/dfuse/bash_fd.yaml

```
@@ -0,0 +1,21 @@
+hosts:
+  test_servers: 1
+timeout: 900
+server_config:
+  name: daos_server
+  engines_per_host: 1
+  engines:
+    0:
+      targets: 4
+      nr_xs_helpers: 0
+      storage:
+        0:
+          class: ram
+          scm_mount: /mnt/daos
+  system_ram_reserved: 1
+pool:
+  scm_size: 1000000000
```

This could be more generic, and could use a human-readable size
```suggestion
  size: 1G
```